HW 04/Task 4.2/RationalNumber.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+namespace Task_4._2
+{
+    public sealed class RationalNumber : IComparable<RationalNumber>
+    {
+        private int _numerator;
+        private int _denominator;
+
+        public RationalNumber(int numerator, int denominator)
+        {
```

Comment:
int denominator = 1, to provide usage of constructor with 1 argument for simple  integer values